src/pages/wishlist.html

```diff
@@ -0,0 +1,193 @@
+                          <p>1</p>
+                          <button class="ml-8 borderless-btn"><i class="fa fa-plus"></i></button>
+                        </div>
+                        <div class="flex-center">
+                        <button class="text-center borderless-btn text-underline">Remove</button>
+                      </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="cart-item-wrapper">
+                    <p class="text-center uppercase">
+                       Raw Bites Premium Dog Treats-Chicken
+                      Jerky
+                    </p>
+                    <div class="cart-item flex align-items-center">
+                      <img src="../images/cart-1.jpg" alt="cart-product" class="small-img" loading="lazy">
+                      <div class="cart-item-details" style="flex: 1;text-align: center;">
+                        <p>Pack of 1</p>
+                        <p>Rs.499/-</p>
+                        <div class="flex-center move-up">
+                          <button class="mr-8 borderless-btn"><i class="fa fa-minus"></i></button>
+                          <p>1</p>
+                          <button class="ml-8 borderless-btn"><i class="fa fa-plus"></i></button>
+                        </div>
+                        <div class="flex-center">
+                        <button class="text-center borderless-btn text-underline">Remove</button>
+                      </div>
+                      </div>
+                    </div>
+                  </div>
+                  <p class="uppercase text-center">Total</p>
+                  <h6 class="text-center move-up">Rs. 998</h6>
+                  <button class="uppercase btn primary-btn checkout-btn">Checkout</button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </nav>
+      </header>
+      <main>
+        <div class="content-wrapper">
+            <h3 class="text-center">My Wishlist <span class="light-txt">(3 items)</span></h3>
+          <section id="wishlist" class="wishlist-wrapper">
+            <div class="card card-vertical products-list-item">
+              <div class="card-media">
+                <img
+                  src="../images/clothes/dgClothes-4.jpg"
+                  alt=""
+                  class="responsive-img"
+                  loading="lazy"
+                />
+              </div>
+              <div class="card-description-wrapper">
+                <div class="card-primary">
+                  <p class="uppercase">Skull Dog Tank Top</p>
+                  <p class="card-primary-subtitle">Rs. 899/-</p>
+                </div>
+                <div class="card-actions">
+                  <button class="btn primary-btn uppercase">Add to cart</button>
+                </div>
+                <button class="borderless-btn ml-8 wishlist-btn"><i class="far fa-heart fa-lg"></i></button>
```

Comment:
Suggestion : as this is wishlist page applying red color to wishlist icon which is in card makes UI self explanatory